app/views/devise/registrations/_profile_form.html.haml

```diff
@@ -0,0 +1,58 @@
+= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
+  = render "devise/shared/error_messages", resource: resource
```

Comment:
This will place all of the errors in an alert at the top of the form. Can you take a look at the logic for the new user sign up form that places the errors messages on their corresponding field?